Descrption: Update preferred applications xml file for banshee to use banshee-1 instead of banshee executable
Bug: bugs.launchpad.net/bugs/533888

Index: gnome-control-center-2.30.1/capplets/default-applications/gnome-default-applications.xml.in
===================================================================
--- gnome-control-center-2.30.1.orig/capplets/default-applications/gnome-default-applications.xml.in	2010-06-23 16:16:51.308187001 +1000
+++ gnome-control-center-2.30.1/capplets/default-applications/gnome-default-applications.xml.in	2010-06-23 16:17:02.858187000 +1000
@@ -297,8 +297,8 @@
   <media-players>
     <media-player>
       <_name>Banshee Music Player</_name>
-      <executable>banshee</executable>
-      <command>banshee</command>
+      <executable>banshee-1</executable>
+      <command>banshee-1</command>
       <icon-name>music-player-banshee</icon-name>
       <run-in-terminal>false</run-in-terminal>
     </media-player>
